oap-logstream/src/main/java/oap/logstream/disk/ParquetWriter.java

```diff
@@ -176,19 +187,24 @@ private void convertToParquet( byte[] buffer, int offset, int length, byte[][] t
         ParquetSimpleGroup group = new ParquetSimpleGroup( messageType );
         Object obj = bis.readObject();
         while( obj != null ) {
+            int parquetCol = 0;
             while( obj != null && obj != BinaryInputStream.EOL ) {
-                var colType = types[col];
-                try {
-                    addValue( col, obj, colType, 0, group );
-                } catch( Exception e ) {
-                    log.error( "header {} class {} type {} col {}", headers[col], obj.getClass().getName(),
-                        Lists.map( List.of( ArrayUtils.toObject( types[col] ) ), oap.template.Types::valueOf ),
-                        col );
-
-                    var data = BinaryUtils.read( buffer, offset, length );
-                    log.error( "object data {}", data );
-
-                    throw e;
+                byte[] colType = types[col];
+                String header = headers[col];
+                if( !excludeFields.contains( header ) ) {
+                    try {
+                        addValue( parquetCol, obj, colType, 0, group );
+                    } catch( Exception e ) {
+                        log.error( "header {} class {} type {} col {}", header, obj.getClass().getName(),
+                            Lists.map( List.of( ArrayUtils.toObject( types[col] ) ), oap.template.Types::valueOf ),
+                            parquetCol );
+
+                        var data = BinaryUtils.read( buffer, offset, length );
+                        log.error( "object data {}", data );
```

Comment:
why not combined with an above log.error?

Reply:
This is an additional log to make troubleshooting easier.